packages/server/src/server.ts

```diff
@@ -43,12 +43,21 @@ const cors = require('koa-cors');
 const app = new Koa();
 const rpc = koaJsonRpc();
 
+const responseSuccessLabel = 'Success';
+const responseFailureLabel = 'Failure';
 const register = new Registry();
 collectDefaultMetrics({ register, prefix: 'rpc_relay_' });
 const methodSuccessLatencyCounter = new Counter({
-  name: 'rpc_method_success_latency_counter',
-  help: 'JSON RPC method success latencycounter',
-  labelNames: ['method', 'success', 'latency'],
+  name: 'rpc_method_success_counter',
```

Comment:
* Shouldn't name it as `success` counter since it tracks both success and non-success and it would be inefficient to have a separate non-success counter.
* Its customary to name counters as ending in `count`.
* Other metrics start with `rpc_relay` so this should too

Suggest `rpc_relay_request_count`